embeddings/model/lightning_module/sequence_labeling.py

```diff
@@ -73,3 +88,64 @@ def test_step(self, *args: Any, **kwargs: Any) -> Optional[STEP_OUTPUT]:
         else:
             _logger.warning("Missing labels for the test data")
         return None
+
+    def str2int(self, values: Union[str, Iterable[Any]]) -> Union[int, Iterable[Any]]:
+        """Conversion class name string => integer duplicated from huggingface ClassLabel."""
```

Comment:
I'm not a big fan of copying functionality from other libs. Why we can't just use wrap ClassLabel object usage?